sdk/servicebus/service-bus/src/util/atomXmlHelper.ts

```diff
@@ -82,7 +82,17 @@ export async function executeAtomXmlOperation(
  */
 export function serializeToAtomXmlRequest(resourceName: string, resource: any): object {
   const content: any = {};
-  content[resourceName] = resource;
+
+  // The top level key value pairs having undefined/null as the value are removed in order to address issue where the Service Bus'
+  // ATOM based management operations throw a "Bad Request" error if empty tags are included in the xml request body at top level.
+  const processedResource = Object.assign({}, resource);
+  Object.keys(processedResource).forEach(function(property) {
+    if (processedResource[property] == undefined) {
+      delete processedResource[property];
+    }
+  });
```

Comment:
Nit: Right now we are looping through all the properties of `resource` twice.
First, by the `Object.assign` in order to make the copy.
Second, by the `Object.keys`

We could make do with a single loop
```javascript
const processedResource = {};
Object.keys(resource).forEach(property => {
    if (resource[property] != undefined) {
      processedResource[property] = resource[property]
    }
  });
```